pkg/server/auth/server_test.go

```diff
@@ -625,6 +633,69 @@ func TestUserInfoOIDCFlow(t *testing.T) {
 	g.Expect(info.Email).To(Equal("jane.doe@example.com"))
 }
 
+func TestRefresh(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	tokenSignerVerifier, err := auth.NewHMACTokenSignerVerifier(5 * time.Minute)
+	g.Expect(err).NotTo(HaveOccurred())
+
+	s, m := makeAuthServer(t, nil, tokenSignerVerifier, []auth.AuthMethod{auth.OIDC})
+
+	tokens := getVerifyTokens(t, s, m)
+
+	tf := tokens["refresh_token"].(string)
+
+	req := httptest.NewRequest(http.MethodGet, "https://example.com/test", nil)
+	req.AddCookie(&http.Cookie{
+		Name:  auth.RefreshTokenCookieName,
+		Value: tf,
+	})
+
+	w := httptest.NewRecorder()
+	cookies := make(map[string]*http.Cookie)
```

Comment:
Nitpick: this could be moved to above line 659 because you don't need it to determine whether or not it succeeds?

From https://wiki.c2.com/?DeclareVariablesAtFirstUse
> It's [good] to declare variables when you first use them to ensure that they are always initialized to some valid value and that their intended use is always apparent.